locksmith/src/controllers/v2/metadataController.ts

```diff
@@ -139,8 +139,7 @@ export class MetadataController {
           returning: true,
         }
       )
-      const statusCode = created ? 201 : 204
```

Comment:
does not return created on sequelize v6 so can't differentiate.

Reply:
ha! that's ok